test/gns.test.ts

```diff
@@ -983,4 +983,68 @@ describe('GNS', () => {
       await gns.connect(me.signer).mintNSignal(me.address, 1, toGRT('10'), 0)
     })
   })
+
+  describe('batch calls', function () {
```

Comment:
looks good. i would like to add a few tests to play it safe. i want to add the following sequences:

- publishSubgraph, publishSubgraph
- publishSubgraph, publishNewVersion
- publishSubgraph, mintNSignal, burnNsignal

I think it will be fine. I can write the tests for these

Reply:
Good idea